sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -729,6 +760,30 @@ public void testBulkInsertWithMultiValue() throws Exception {
     }
 
     @Test
+    public void testBulkInsertWithMultiValueFailing() throws Exception {
```

Comment:
This is a bulk insert with multivalues. We have one with multivalues but not bulk and one with bulk but not multivalues. Do I miss something?